vue/components/ui/molecules/progress-bar/progress-bar.vue

```diff
@@ -77,6 +81,11 @@ export const ProgressBar = {
             };
             return base;
         },
+        progressBarStyle() {
+            const base = {};
+            if (this.backgroundColor) base.backgroundColor = this.backgroundColor;

```

Comment:
@NFSS10 we should use camel case for CSS properties when setting them using a dict, that's the canonical Vue way